libraries/consentManagement/cmUtils.js

```diff
@@ -1,6 +1,5 @@
 import {timedAuctionHook} from '../../src/utils/perfMetrics.js';
 import {isNumber, isPlainObject, isStr, logError, logInfo, logWarn} from '../../src/utils.js';
-import {ConsentHandler} from '../../src/consentHandler.js';
```

Comment:
this causes a warning on line 48 below

Reply:
yes, but removing these introduces errors. there's a limit to how much I'm prepared to genuflect for a linter - if/when we get to actual type checking is when I'll figure out how to satisfy it

Reply:
fair enough